ironfish/src/network/types.ts

```diff
@@ -18,7 +18,11 @@ export enum NetworkMessageType {
   PeerListRequest = 10,
   Signal = 11,
   SignalRequest = 12,
-  NewBlockHashes = 13,
+  PooledTransactionsRequest = 13,
+  PooledTransactionsResponse = 14,
+  NewPooledTransactionHashes = 15,
+  NewTransactionV2 = 16,
+  NewBlockHashes = 17,
```

Comment:
You'll probably have to resolve this with the messages I just merged to staging -- I don't care who gets which numbers 😆

Reply:
damn thought I'd get in there first. shouldn't have approved your PR haha

Reply:
haha, that was your fatal mistake 😆